h2/service/wrapper.conf

```diff
@@ -1,101 +1,101 @@
-#wrapper.java.initmemory=16
-
-# Maximum Java Heap Size (in MB)
-#wrapper.java.maxmemory=64
-
-# Application parameters.  Add parameters as needed starting from 1
-wrapper.app.parameter.1=org.h2.tools.Server
-wrapper.app.parameter.2=-tcp
-wrapper.app.parameter.3=-web
-
-#********************************************************************
-# Wrapper Logging Properties
-#********************************************************************
-# Format of output for the console.  (See docs for formats)
-wrapper.console.format=PM
-
-# Log Level for console output.  (See docs for log levels)
-wrapper.console.loglevel=INFO
-
-# Log file to use for wrapper output logging.
-wrapper.logfile=wrapper.log
-
-# Format of output for the log file.  (See docs for formats)
-wrapper.logfile.format=LPTM
-
-# Log Level for log file output.  (See docs for log levels)
-wrapper.logfile.loglevel=INFO
-
-# Maximum size that the log file will be allowed to grow to before
-#  the log is rolled. Size is specified in bytes.  The default value
-#  of 0, disables log rolling.  May abbreviate with the 'k' (kb) or
-#  'm' (mb) suffix.  For example: 10m = 10 megabytes.
-wrapper.logfile.maxsize=0
-
-# Maximum number of rolled log files which will be allowed before old
-#  files are deleted.  The default value of 0 implies no limit.
-wrapper.logfile.maxfiles=0
-
-# Log Level for sys/event log output.  (See docs for log levels)
-wrapper.syslog.loglevel=NONE
-
-#********************************************************************
-# Wrapper Windows Properties
-#********************************************************************
-# Title to use when running as a console
-wrapper.console.title=H2 Console
-
-#********************************************************************
-# Wrapper Windows NT/2000/XP Service Properties
-#********************************************************************
-# WARNING - Do not modify any of these properties when an application
-#  using this configuration file has been installed as a service.
-#  Please uninstall the service before modifying this section.  The
-#  service can then be reinstalled.
-
-# Name of the service
-wrapper.ntservice.name=H2DatabaseService
-
-# Display name of the service
-wrapper.ntservice.displayname=H2 Database Engine Service
-
-# Description of the service
-wrapper.ntservice.description=H2 Database Engine Service (TCP Server and H2 Console Web-Application)
-
-# Service dependencies.  Add dependencies as needed starting from 1
-wrapper.ntservice.dependency.1=
-
-# Mode in which the service is installed.  AUTO_START or DEMAND_START
-wrapper.ntservice.starttype=AUTO_START
-
-# Allow the service to interact with the desktop.
-wrapper.ntservice.interactive=false
-
-# wrapper.ntservice.account=
-# wrapper.ntservice.password=  
+#********************************************************************
+# Wrapper Properties
+#********************************************************************
+# Java Application
```

Comment:
You can't add this configuration file that easy. You need to adjust generation of Windows and ZIP distributions too, because they are currently packaged with another wrapper. It's better to do it in a separate pull request.

See also #2410.

Reply:
Sorry, I make some mistakes duriing merge. This file should not be commited ;-(

I will try rebase and double check before push.